docs/multi-instance.md

```diff
@@ -8,13 +8,89 @@ See [554#issuecomment-496804929](https://github.com/elastic/ansible-elasticsearc
 
 If you have single-instances hosts and want to upgrade from previous versions of the role:
 
+### Procedure with data move
+
+This procedure will allow you to move your data to the new standard paths (see [#581](https://github.com/elastic/ansible-elasticsearch/issues/581)):
+
+1. Stop Elasticsearch before the migration
+
+2. Migrate your data to the new standard paths:
+```
+# mv /etc/elasticsearch/${ES_INSTANCE_NAME}/* /etc/elasticsearch/ && rm -fr /etc/elasticsearch/${ES_INSTANCE_NAME}/
+mv: overwrite '/etc/elasticsearch/elasticsearch.keystore'? y
+# mv /var/lib/elasticsearch/${INVENTORY_HOSTNAME}-${ES_INSTANCE_NAME}/* /var/lib/elasticsearch/ && rm -fr /var/lib/elasticsearch/${INVENTORY_HOSTNAME}-${ES_INSTANCE_NAME}/
+# ls /var/lib/elasticsearch/
+nodes
+# mv /var/log/elasticsearch/${INVENTORY_HOSTNAME}-${ES_INSTANCE_NAME}/* /var/log/elasticsearch/ && rm -fr /var/log/elasticsearch/${INVENTORY_HOSTNAME}-${ES_INSTANCE_NAME}/
+# rm -fr /var/run/elasticsearch/${INVENTORY_HOSTNAME}-${ES_INSTANCE_NAME}/
+```
+
+3. Update playbook (remove `es_conf_dir`, `es_data_dirs`, `es_log_dir`, `es_pid_dir` and `es_instance_name` variables)
+
+4. Deploy ansible-role
```

Comment:
Perhaps explicitly call out to use the new ansible role hre

Reply:
Ansible Galaxy command to install the new version of the role is in the example below: https://github.com/elastic/ansible-elasticsearch/pull/605/files#diff-05d70e203a6d81fe483f4556abe6750dR44

Is it what you expect?

Reply:
Yes the example is what I expect, but there is not a step in the instructions that mentions updating the role or using the new role in step 4 here.

Reply:
fixed in https://github.com/elastic/ansible-elasticsearch/pull/605/commits/fb64bd7d1cd88e858614522d84e11dffddfc8041